### DISCUSSION TITLE
Is it possible to limit crowdstrike-spotlight-slacker to test group so it does not spam the whole scope?

### DISCUSSION THREAD

**Discussion by katruhanchik** [hazcod/crowdstrike-spotlight-slacker#6] [Q&A] [answered]
Hi! I'm currently trying to figure out how to implement this. Is it possible to limit triggering of alerts to specific hosts or host groups for test purposes and spam limitation? Or limit triggers by severity with only proven existing exploits or active use of the vulnerability?

**Comment by hazcod** [comment, marked as answer]
@katruhanchik currently it will send a Slack message to the users whenever an update is available for that user for every vulnerability.
Filtering for exploits could be a nice feature!

**Reply by hazcod** [reply]
Hi @katruhanchik , see https://github.com/hazcod/security-slacker/releases/tag/v1.6.0 for the possibility to filter on Falcon results with `severities` and `CVE base score`!
